### PR TITLE
Update init to allow suppression

### PR DESCRIPTION
**Pull request by naddeoa**
The new init system can make for confusing demos for existing users and new use cases. Making this suppression option to allow us to avoid output when we don't need to see it.

This is also the first example that was updated to use init, though its a kind of weird case since we don't actually want to display the output in this context.

### CODE REVIEW

File: python/examples/integrations/writers/Single_Image_Tracing_Profile_to_WhyLabs.ipynb
Comment (FelipeAdachi) [inline python/examples/integrations/writers/Single_Image_Tracing_Profile_to_WhyLabs.ipynb:1]: Is feast a dependency for the required extras?
if not, maybe it's from your environment, @naddeoa ?

In any case, maybe we could clear the output from install whylogs

Comment (jamie256) [inline python/examples/integrations/writers/Single_Image_Tracing_Profile_to_WhyLabs.ipynb:1]: This is useful to demonstrate the changes, but this example was a side branch I don't think we want to check this file in as a main example.

**Review by jamie256 — CHANGES_REQUESTED** [review]
Improvement over our current output, thanks!

Let's remove that test example notebook from the PR, otherwise looks good!

File: python/examples/integrations/writers/Single_Image_Tracing_Profile_to_WhyLabs.ipynb
Comment (naddeoa) [reply python/examples/integrations/writers/Single_Image_Tracing_Profile_to_WhyLabs.ipynb:1]: This one isn't the big doc update. This pr was for a one off to get Felipe's example working with init. The other PR shows init better: https://github.com/whylabs/whylogs/pull/1365

File: python/test_notebooks/notebook_tests.py [hunk 1]
```diff
@@ -8,6 +8,7 @@
 OUTPUT_NOTEBOOK = "output.ipynb"
 skip_notebooks = [
     "Guest Session.ipynb",
+    "Single_Image_Tracing_Profile_to_WhyLabs.ipynb",
```
Comment (jamie256) [inline python/test_notebooks/notebook_tests.py:11]: don't need this now that the notebook file is removed.

**Review by jamie256 — APPROVED** [review]
LGTM!

**Pull request merged**